<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>URW Bookman</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Black Jack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/картинки курсач/Фон 1_16-9.png);</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>835</x>
        <y>520</y>
        <width>250</width>
        <height>230</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <pointsize>29</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Загрузки/Telegram Desktop/image_9_1.png</normaloff>../Загрузки/Telegram Desktop/image_9_1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>750</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <pointsize>29</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../Загрузки/Telegram Desktop/exit_1.png</normaloff>../Загрузки/Telegram Desktop/exit_1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>785</x>
        <y>100</y>
        <width>350</width>
        <height>350</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Загрузки/Telegram Desktop/image_4_1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/картинки курсач/Фон 1_16-9.png);</string>
     </property>
     <widget class="QWidget" name="pauseMenu" native="true">
      <property name="geometry">
       <rect>
        <x>755</x>
        <y>315</y>
        <width>410</width>
        <height>450</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent
</string>
      </property>
      <widget class="QPushButton" name="quitGame">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>210</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Загрузки/Telegram Desktop/exit_1.png</normaloff>../Загрузки/Telegram Desktop/exit_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="resume">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>210</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Загрузки/Telegram Desktop/continue_1.png</normaloff>../Загрузки/Telegram Desktop/continue_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/картинки курсач/Фон 1_16-9.png);</string>
     </property>
     <widget class="QWidget" name="levelchoosing" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>545</x>
         <y>250</y>
         <width>830</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>URW Bookman</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent</string>
       </property>
       <property name="text">
        <string>Выберите количество игроков</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="easyDiff">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>550</y>
         <width>300</width>
         <height>300</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../картинки курсач/Кнопки игроков-20240515T124431Z-001/Кнопки игроков/1 игрок.png</normaloff>../картинки курсач/Кнопки игроков-20240515T124431Z-001/Кнопки игроков/1 игрок.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="mediumDiff">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>550</y>
         <width>300</width>
         <height>300</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../картинки курсач/Кнопки игроков-20240515T124431Z-001/Кнопки игроков/2 игрока.png</normaloff>../картинки курсач/Кнопки игроков-20240515T124431Z-001/Кнопки игроков/2 игрока.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="hardDiff">
       <property name="geometry">
        <rect>
         <x>1150</x>
         <y>550</y>
         <width>300</width>
         <height>300</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../картинки курсач/Кнопки игроков-20240515T124431Z-001/Кнопки игроков/3 игрока.png</normaloff>../картинки курсач/Кнопки игроков-20240515T124431Z-001/Кнопки игроков/3 игрока.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="backFromLevelMenu">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Загрузки/Telegram Desktop/image_8_1.png</normaloff>../Загрузки/Telegram Desktop/image_8_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/картинки курсач/Стол 5.3.png);
    background-repeat: no-repeat;
    background-position: center;
</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QPushButton" name="pauseButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>111</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Загрузки/Telegram Desktop/image_7_1.png</normaloff>../Загрузки/Telegram Desktop/image_7_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="betButton">
       <property name="geometry">
        <rect>
         <x>915</x>
         <y>980</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Bet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="doubleUp">
       <property name="geometry">
        <rect>
         <x>695</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Double up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="repeatButton">
       <property name="geometry">
        <rect>
         <x>1135</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
      <widget class="QPushButton" name="oneKButton">
       <property name="geometry">
        <rect>
         <x>805</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>1K</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fiveKButton">
       <property name="geometry">
        <rect>
         <x>915</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>5K</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tenKButton">
       <property name="geometry">
        <rect>
         <x>1025</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>10K</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>950</y>
         <width>100</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent</string>
       </property>
      </widget>
      <widget class="QPushButton" name="hitButton">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Hit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="standButton">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Stand</string>
       </property>
      </widget>
      <widget class="QPushButton" name="splitButton">
       <property name="geometry">
        <rect>
         <x>1070</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Split</string>
       </property>
      </widget>
      <widget class="QPushButton" name="doubleButton">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>920</y>
         <width>90</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Double</string>
       </property>
      </widget>
      <widget class="QPushButton" name="splitStand">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>600</y>
         <width>88</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Stand</string>
       </property>
      </widget>
      <widget class="QPushButton" name="splitHit">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>630</y>
         <width>88</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Hit</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
